README.md

```diff
@@ -41,7 +41,7 @@ Let's start with an example that shows the sessions API in a nutshell:
 		session.Values["foo"] = "bar"
 		session.Values[42] = 43
 		// Save it before we write to the response/return from the handler.
-		err = session.Save(r, w)
+		err := session.Save(r, w)
```

Comment:
It might be better for us to handle the error in the example above instead of ignoring it, since many folks copy-paste this code

Reply:
The example has already handled the error:

```go
if err != nil {
	http.Error(w, err.Error(), http.StatusInternalServerError)
	return
}
```

This patch fixes typo, since `err` is undefined.